go.mod

```diff
@@ -293,3 +293,8 @@ require (
 )
```

Comment:
Do we need this change? operator doesn't consume the package. And the pkg containing the vulnerabilities `github.com/cert-manager/cert-manager` is used in operator mainly for API defs and doesn't invoke any functionality involving jwt package, correct?